doc/modules/classes.rst

```diff
@@ -674,6 +674,7 @@ Kernels
 
    inspection.partial_dependence
    inspection.permutation_importance
+   inspection.metric_threshold_curve
```

Comment:
I don't think that we should include this feature in this module. Actually, as the precision-recall and roc curves, this is a curve metric. Also, by including it in `sklearn.metrics` module, we can drop the `metric_` naming.

I would therefore call it `sklearn.metrics.decision_threshold_curve`